src/buildCommon.js

```diff
@@ -423,7 +505,7 @@ const makeVerb = function(group, options) {
 
 // A map of spacing functions to their attributes, like size and corresponding
 // CSS class
-const spacingFunctions = {
+const spacingFunctions: {[string]: {| size: string, className: string |}} = {
```

Comment:
I was confused for a second until I realize that `[string]` was the key.

Reply:
One thing we might consider doing is to switch from `string[]` to `Array<string>` for arrays.  I've opened https://github.com/Khan/KaTeX/issues/987 to discuss it.